src/keywords.YAML-tmLanguage

```diff
@@ -20,7 +20,7 @@ repository:
                 match: \b(data|inline|tailrec|operator|infix|typealias|reified)\b
                 name: storage.type.kotlin
             -
-                match: \b(external|public|private|protected|internal|abstract|final|sealed|enum|open|annotation|override|vararg|typealias|expect|actual|suspend|yield|out|in|lateinit)\b
+                match: \b(external|public|private|protected|internal|abstract|final|sealed|enum|open|annotation|override|vararg|typealias|expect|actual|suspend(?!\s*[\(]?\s*\{)|yield|out|in|lateinit)\b
```

Comment:
We could probably break this out into its own line to make it easier to maintain